applications/asset_tracker_v2/README.rst

```diff
@@ -377,13 +377,15 @@ The application contains examples of Kconfig overlays.
 The following configuration files are available in the application folder:
 
 * :file:`prj.conf` - Configuration file common for all build targets
-* :file:`boards/thingy91_nrf9160ns.conf` - Configuration file specific for Thingy:91. The file is automatically merged with :file:`prj.conf` when you build for the ``thingy91_nrf9160ns`` build target.
+* :file:`boards/thingy91_nrf9160ns.conf` - Configuration file specific for Thingy:91. This file is automatically merged with the :file:`prj.conf` file when you build for the ``thingy91_nrf9160ns`` build target.
+* :file:`boards/nrf9160dk_nrf9160ns.conf` - Configuration file specific for nRF9160 DK. This file is automatically merged with the :file:`prj.conf` file when you build for the ``nrf9160dk_nrf9160ns`` build target.
 * :file:`overlay-low-power.conf` - Configuration file that achieves the lowest power consumption by disabling features  that consume extra power like LED control and logging.
 * :file:`overlay-debug.conf` - Configuration file that adds additional verbose logging capabilities to the application
+* :file:`boards/<BOARD>/led_state_def.h` - Header file that describes the LED behavior of the CAF LEDs module.
 
 Generally, Kconfig overlays have an ``overlay-`` prefix and a ``.conf`` extension.
 Board-specific configuration files are placed in the :file:`boards` folder and are named as :file:`<BOARD>.conf`.
-DTS overlay files are named the same as the build target and use the file extension ``.overlay``.
+DTS overlay files are named the same as the build target and use the file extension ``.overlay``. They are placed in the :file:`boards/<BOARD>.overlay>` folder.
```

Comment:
```suggestion
DTS overlay files are named the same as the build target and use the file extension ``.overlay``. They are placed in the :file:`/boards` folder.
```
Sorry, once more about this. Is't the folder name just `boards`?

Reply:
yes, sorry changed file locations but did not changed it here